src/main/java/nl/avans/drivio/controller/UserController.java

```diff
@@ -1,45 +1,42 @@
 package nl.avans.drivio.controller;
 
-import nl.avans.drivio.service.IUserManager;
 import nl.avans.drivio.model.User;
+import nl.avans.drivio.repository.UserRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
+import java.util.Optional;
 
 @RestController
-@RequestMapping(path = "/drivio")  // Optional
+@RequestMapping("/drivio/user")
 public class UserController {
 
-    private final IUserManager userManager;
+    private final UserRepository userRepository;
 
     @Autowired
-    public UserController(IUserManager userManager) {
-        this.userManager = userManager;
+    public UserController(UserRepository userRepository) {
+        this.userRepository = userRepository;
     }
 
-    @GetMapping(path = "/users")
+    @GetMapping("/users")
     public List<User> getAllUsers() {
-        return this.userManager.findAll();
+        return (List<User>) userRepository.findAll();
     }
 
-    @GetMapping(path = "/users/{id}")              // Curly braces means 'expecting an id arg'
-    public User findById(@PathVariable int id) {     // Get arg from body. Example: /api/users/1
-        return this.userManager.findById(id);
+    @GetMapping("/users/{id}")
+    public Optional<User> getById(@PathVariable int id) {
+        return userRepository.findById(id);
     }
 
-    @PostMapping(path = "/add")
-    public void add(@RequestBody User user) {        // Arg will be given from the body of the request (postman)
-        this.userManager.add(user);
+    @PostMapping("/add")
```

Comment:
Je hebt hier path "/add" gebruikt, in de andere code hebben we niet overal een path gedefineerd. We moeten goed kijken wat we gaan gebruiken om de code in een lijn te hebben

Reply:
Paths aangepast in commit: https://github.com/leonvsc/Drivio/pull/26/commits/b3e0b32306df102667a7056ca493e0b9997b8e4f